packages/block-library/src/navigation/editor.scss

```diff
@@ -271,20 +280,8 @@ $color-control-label-height: 20px;
 	}
 
 	&:not(.is-loading) {
-		// Don't show the preview boxes for an empty nav block,
-		// but be technically present to help size the empty state.
+		// Don't show the preview boxes for an empty nav block.
```

Comment:
What are the "preview boxes"? Anyway to be more specific here to add clarity?

Reply:
Oh good one! That actually suggests this needs a quick look again after the rebase! I'll take a look, thank you!